src/System.Text.Json/tests/Serialization/EnumTests.cs

```diff
@@ -2,7 +2,9 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+using System.Collections.Generic;
```

Comment:
nit: Are these new using directives needed?

Reply:
Yes used of IEnumerable that provides test data, e.g. JsonWithInvalidOutOfBoundNumber

Reply:
I broke down tests to multiple rather than using a single MemberData. now I won't need this new using directive